src/app/organisms/emoji-board/recent.js

```diff
@@ -0,0 +1,31 @@
+import initMatrix from '../../../client/initMatrix';
+import { emojis } from './emoji';
+
+const eventType = 'io.element.recent_emoji';
+
+function get() {
+  return initMatrix.matrixClient.getAccountData(eventType).getContent().recent_emoji ?? [];
+}
+
+export function getRecentEmojis(limit) {
+  return get()
+    .sort((a, b) => b[1] - a[1])
+    .slice(0, limit)
+    .map(([unicode]) => emojis.find((e) => e.unicode === unicode));
```

Comment:
you are using `map` here, not sure if you are intended to filter it? cuz custom emojis crash the app when EmojiBoard opens

Reply:
Now that fixed the custom emojis being added, should I still filter it?

Reply:
is it mapped to get emojidata?

Reply:
we can just filter it for now. we will improve things later when working specifically on emojiboard

Reply:
Not sure if there's still a need to filter now as custom emojis are not being added anymore. Should I still filter just to be sure?

Reply:
yeah, as cinny is not the only client which can update that :)